### PR TITLE
Glance json feed

### PR DESCRIPTION
**Pull request by mparent61**
@spra85 @MichaelButkovic @benghaziboy 

New ingest feed for Glance

Based on https://s3.amazonaws.com/onionwebtech/sample-glance-onion.json

### CODE REVIEW

File: bulbs/feeds/views.py [hunk 1]
```diff
@@ -56,3 +64,20 @@ def get_queryset(self):
...
+
+
+class GlanceFeedViewSet(viewsets.ReadOnlyModelViewSet):
+
+    model = Content
+    serializer_class = GlanceContentSerializer
+
+    queryset = Content.search_objects.search().sort('-last_modified').filter(Published())
+
+    permission_classes = [AllowAny]
```
Comment (MichaelButkovic) [inline bulbs/feeds/views.py]: `permission_classes` is usually a tuple, so this would be `permission_classes=(AllowAny,)`

**Comment by MichaelButkovic** [comment]
LGTM👍

File: bulbs/feeds/views.py (hunk 1 above)
Comment (mparent61) [inline bulbs/feeds/views.py]: Ah, thx

**Comment by spra85** [comment]
👍

**Pull request merged**